partner_product_price/data/ir_actions_server.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="action_product_price_list_report" model="ir.actions.server">
+        <field name="name">Products</field>
+        <field name="groups_id" eval="[(4, ref('resource_booking.group_user'))]" />
```

Comment:
this module ``resource_booking`` is not set in the dependency field of the manifest.